browser/brave_account/brave_account_service_factory.cc

```diff
@@ -34,6 +34,10 @@ BraveAccountServiceFactory::BraveAccountServiceFactory()
 
 BraveAccountServiceFactory::~BraveAccountServiceFactory() = default;
 
+bool BraveAccountServiceFactory::ServiceIsCreatedWithBrowserContext() const {
+  return true;
```

Comment:
Do we create the service at startup so we can continue polling in the event that the user restarts the browser during verification?